### PR TITLE
Improvises the documentation to migrate neetoUI to v4

### LINKED ISSUE #1332: Add a command to upgrade neeto-ui version to v4 in neetoui
@yedhink

### PR DESCRIPTION
**Pull request by deepakjosp**
Fixes #1332 

**Description**
Changed documentation to migrate neetoUI to v4.

**Checklist**

- [x] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary label (patch/minor/major - If package publish is required)
- [x] I have followed the suggested description format and styling

**Reviewers**

### CODE REVIEW

**Comment by deepakjosp** [comment]
@karthiknmenon  _a could you please review this PR?

**Review by karthiknmenon — APPROVED** [review]
@amaldinesh7 _a Please confirm and merge.

File: stories/Migration-Guide/v3tov4.stories.mdx [hunk 1]
```diff
@@ -18,13 +18,31 @@ This document will help you upgrade from neetoUI `3.x` version to neetoUI `4.x`
...
+2. **Commit the changes** to `package.json` and `yarn.lock` immediately.
+
+```shell
+git commit -m "Upgraded neetoUI to v4"
+```
+
+3. As we have removed the dependencies from the neetoUI package, you will have to install them separately.
+
+```shell
+yarn add @emotion/is-prop-valid@1.2.0 antd@4.23.6
```
Comment (amaldinesh7) [inline stories/Migration-Guide/v3tov4.stories.mdx:34]: Please keep antd only. @emotion/is-prop-valid@1.2.0 is not needed.

**Review by amaldinesh7 — APPROVED** [review]
@deepakjosp _a Minor comment. Please proceed with merging once fixed.

**Pull request merged**